tools/cli/commands/changelog.js

```diff
@@ -221,6 +243,20 @@ export async function changelogValidate( argv ) {
 	changeloggerCli( argv );
 }
 
+/**
+ * Changelog write script.
+ *
+ * @param {object} argv - arguments passed to the CLI.
+ */
+async function changelogWrite( argv ) {
+	argv = await validateProject( argv );
+	const args = process.argv.slice( 5 );
```

Comment:
I note this fails oddly if you put args before the project name, e.g. `jetpack changelog write --use-version=100.0.0 plugins/jetpack`

Reply:
Redid this a bit. Should be fixed in latest commit.